layouts/shortcodes/integration-assets-reference.md

```diff
@@ -3,7 +3,7 @@ You can find the complete list of mandatory and optional attributes for the `man
 | Attribute                                            | Mandatory or Optional       | Description                                                                                                                                                                                                                                                                       |
 |----------|----------|----------|
 | `manifest_version`                                   | Mandatory                   | **Type**: String Enum.<br><br>Version of the manifest schema. Supported values include `1.0.0` and `2.0.0`.                                                                                                                                                                       |
-| `app_uuid`                                           | Mandatory                   | **Type**: UUID.<br><br>Globally unique UUID for this application.                                                                                                                                                                                                                 |
+| `app_uuid`                                           | Mandatory                   | **Type**: UUID.<br><br>Globally unique UUID for this application. This is automatically generated by Datadog when running the `ddev create` command during the process of creating your tile scaffolding for your integration tile.                                                                                                                                                                                                                   |
```

Comment:
Can we link https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#populate-the-integration-tile-scaffolding  to ddev create?